### ISSUE TITLE
Readonly setting within SQL Backend options should prevent changes to Connections

### ISSUE THREAD

**Issue by ali-avci** [mRemoteNG/mRemoteNG#2225]
Beside the requirement that when something is changed in the sql-backend options a restart has to be done. It is also required that none of the configurations might be changed after the read-only flag is set in the options. 

It turned out that a user in a shared jump server had changed some connection configuration data and the tool did save them in the normal .xml file in a faulty way. To prevent such attractions it is needed to lock the complete connection configuration when the read-only flag is set, or if the user is a readonly database user (might be hard to identify).

mRemoteNG 1.77.1 with mysql backend.

**Comment by Kvarkas** [comment]
yes that planed to implement with user profiles - then it will be possible to set profile for a user defining what server settings he can update for example and which he just use and not save if any changes are made.

**Comment by robertpopa22** [comment]
**Fix available for testing**

A fix for this issue has been implemented in our community fork (`robertpopa22/mRemoteNG`, branch `main`). A beta build including this fix is available from the fork's Releases page.

Please test and report if this resolves your issue.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
